src/Serilog.AspNetCore/AspNetCore/RequestLoggingOptions.cs

```diff
@@ -50,6 +52,13 @@ public class RequestLoggingOptions
         /// </summary>
         public Action<IDiagnosticContext, HttpContext> EnrichDiagnosticContext { get; set; }
 
+
+        /// <summary>
+        /// The logger through which request completion events will be logged. The default is to use the
+        /// static <see cref="Log"/> class.
+        /// </summary>
+        public ILogger Logger { get; set; }
```

Comment:
https://github.com/serilog/serilog-aspnetcore/blob/dev/src/Serilog.AspNetCore/SerilogWebHostBuilderExtensions.cs#L146

I think it is worth to use DI provided features. Why to manually specify logger if it is already exists ? Just add new ctor param:
```c#
 public RequestLoggingMiddleware(RequestDelegate next, DiagnosticContext diagnosticContext, RequestLoggingOptions options, ILogger logger)
```

Reply:
This won't work if serilog ILogger is not registered for DI and somneone is only using Log.Logger everywhere - which i may be wrong, but is a valid style for using serilog.

Reply:
Well, then we can use second RequestLoggingMiddleware ctor with ILogger property inside options to satisfy both categories.

Reply:
Thanks for the feedback @sungam3r - will give it some thought :+1: